pages/docs/user-guide/compile-kernel.mdx

```diff
@@ -0,0 +1,49 @@
+# Compiling Linux kernel
+
+Lambdo uses a Linux kernel compiled from source. This is a guide to compile the
+kernel.
+
+## Install dependencies
+
+#### Debian/Ubuntu
+
+```bash copy filename="bash"
+sudo apt update
+sudo apt install git build-essential bc flex bison
+```
+
+#### Fedora/RHEL/CentOS
+
+```bash copy filename="bash"
+sudo dnf install git gcc make xz bc flex bison diffutils
+```
+
+#### Arch Linux
+
+```bash copy filename="bash"
+sudo pacman -Syu
+sudo pacman -S git gcc make xz bc flex bison diffutils
+```
+
+## Build the kernel
+
+Clone the Linux kernel official repository:
+
+```bash copy filename="bash"
+git clone https://github.com/torvalds/linux.git
```

Comment:
the github is not the "official" repository per se, the offical one is git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git

Anyway for our use case it's recommended to download an archive of a tagged version , eg https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.6.tar.xz, so you don't have to download the entire tree.